### PR TITLE
Tc 3.0.5

### PR DESCRIPTION
**Pull request by f-peverali**
# Version Upgrade Template

Version: TC 3.0.5

Date:

## Description

This is a Pullreuqest that requires an increase in the Version number. Therefore, multiple outside-github, related Task have to be performed and checked.

All jobs with an `x` in the boxes were performed to the best of knowledge.

## Pre-Merge Activities

- [x] This PR refers to a versioned Branch with a name and a version number in the form of N.n.n, e.g. "TC_3.2.1".
- [x] This PR has a clean meaningful committ history. Minor committs or committs without descirption have been squashed, at the latest now.
- [x] The _./github/workflows/main.yml_ refers to the correct Firetly Terminal and SUSHI Version. 
    > **_Firely Terminal Pipeline_**  0.4.0.

    > **_SUSHI Versions_**  3.5.0.
- [x] By running the _Release_Publish.py_ script, release version and date was updated accordingly. The script ran without errors.
- [x] Eventually, increase the dependency of to newer Basis Modul (package and sushi-config)
- [x] New Release Notes were created, aglined to the committ history and cleaned. In Github, go to 
  - [x]  _-> Releases_ then _-> Draft a new release_ with the _Modul Name and Version_, then
  - [x] _-> Target the main-Branch_ and _->enter a new Tag according to the Version_, then click.
  - [x] Click _-> Generate Release notes_ , _->Adjust them if necessary_ and _-> Copy/Paste the Details in the RealeaseNotes.md_ of the very Branch you want to merge.
  - [x] Finally _-> Save as Draft_

## Merge and Publishing

- [x] With the updated Version, Dates, and Release Notes (as described above) with the last committ into the Branch you want to merge.
- [x] In GitHub _-> Actions_ the _->CI (FHIR Validation)_ workflow terminates successfully.
- [x] Add the Approve / the PR gets posivitly reviewed by a collegue.
- [x] Merge (without squash) the PR, delete the Branch.

## Post-Merge Activities

- [x] Go to the corresponding SIMPLIFIER Project and _-> Github -> Reimport_ the project.
- [x] Go to the corresponding SIMPLIFIER Project and _-> Packages -> Expand the Dropdown for Create -> Create new package_ for the project.
  - [x] With the corresponding version number, and
  - [x] The Release notes (from above) and a compare-link to the previous Release.
  - [x] Unlist the old package by _-> clicking on the old package_, _-> go to Admininstration_ and _-> click on Unlist_
- [x] Publish the previosuly drafteted Release, including version number, on GitHub.
- [x] Provide / Archive the IG in the corresponding _gh-pages_ branch of the GitHub project.
  - [x] Checkout the Branch (no need to merge it later).
  - [x] Export from Simplifier via _-> Guides -> Expand the Modul ... -> Export_
  - [x] Unpack the zip, remove the packages folder (because its kinda big), and move everything else to a (version coressponding) new folder in the branch folder structure.
  - [x] committ the branch.

### CODE REVIEW

File: Resources/input/fsh/ISiKKalender.fsh [hunk 1]
```diff
@@ -8,8 +8,16 @@ Id: ISiKKalender
...
-* specialty[Fachrichtung] from $authorSpecialtyVS (required)
+* specialty contains 
+  Fachrichtung 0..1 MS and 
+  Fachspezialisierung 0..1 and 
+  ErweiterterFachabteilungsschluessel 0..1
+* specialty[Fachrichtung] from $IHEpracticeSettingVS (required)
+  * ^comment = "Die Wahl des hinterlegten ValueSets (http://ihe-d.de/ValueSets/IHEXDSpracticeSettingCode) wurde mit einem Mitglied der IHE Deutschland Arbeitsgruppe XDS ValueSets (https://www.ihe-d.de/projekte/xds-value-sets-fuer-deutschland/) abgestimmt (Stand:13.06.2024)."
+* specialty[Fachspezialisierung] from $authorSpecialtyVS (required)
+  * ^comment = "Dieses Slice SOLL NICHT genutzt werden und ist nur aufgrund der Kompatibilität beibehalten worden (Stand:13.06.2024)."
+* specialty[ErweiterterFachabteilungsschluessel] from $FachabteilungsschluesselErweitertCS (required)
```
Comment (f-peverali) [inline Resources/input/fsh/ISiKKalender.fsh]: issue: hier VS statt CS

File: Resources/input/fsh/ISiKMedizinischeBehandlungseinheit.fsh [hunk 2]
```diff
@@ -8,14 +8,22 @@ Id: ISiKMedizinischeBehandlungseinheit
...
-* specialty[Fachrichtung] from $authorSpecialtyVS (required)
+* specialty contains 
+  Fachrichtung 0..1 MS and 
+  Fachspezialisierung 0..1 and 
+  ErweiterterFachabteilungsschluessel 0..1
+* specialty[Fachrichtung] from $IHEpracticeSettingVS (required)
+  * ^comment = "Die Wahl des hinterlegten ValueSets (http://ihe-d.de/ValueSets/IHEXDSpracticeSettingCode) wurde mit einem Mitglied der IHE Deutschland Arbeitsgruppe XDS ValueSets (https://www.ihe-d.de/projekte/xds-value-sets-fuer-deutschland/) abgestimmt (Stand:13.06.2024)."
+* specialty[Fachspezialisierung] from $authorSpecialtyVS (required)
+  * ^comment = "Dieses Slice SOLL NICHT genutzt werden und ist nur aufgrund der Kompatibilität beibehalten worden (Stand:13.06.2024)."
+* specialty[ErweiterterFachabteilungsschluessel] from $FachabteilungsschluesselErweitertCS (required)
```
Comment (f-peverali) [inline Resources/input/fsh/ISiKMedizinischeBehandlungseinheit.fsh]: issue : s.o.

Comment (f-peverali) [reply Resources/input/fsh/ISiKMedizinischeBehandlungseinheit.fsh]: todo: das ist nicht der letzte Stand, daher nochmal auf basis von Stufe 2 - letzter Stand

File: ImplementationGuide/markdown/ReleaseNotes.md [hunk 3]
```diff
@@ -4,6 +4,14 @@ Im Rahmen der ISiK-Veröffentlichungen wird das [Semantic Versioning](https://se
 
 Die erste Ziffer X bezeichnet ein Major-Release und regelt die Gültigkeit von Releases. Die dritte Ziffer Y (Release x.0.y) bezeichnet eine technische Korrektur und versioniert kleinere Änderungen (Packages) während eines Jahres, z. B. 1.0.1.
 
+Version: 3.0.5
+
+Datum: tbd.
+
+* Update der IHE ValueSets zu "practiceSetting" in ISiKTermin, ISiKKalender und ISiK MedizinischeBehandlungseinheit (Analog zu Stufe 2) https://github.com/gematik/spec-ISiK-Terminplanung/pull/199/commits/7218c4f9f085604e311761d5e76647d09aea04d1
```
Comment (f-peverali) [inline ImplementationGuide/markdown/ReleaseNotes.md]:
```suggestion
* Update der IHE ValueSets zu "practiceSetting" in ISiKTermin, ISiKKalender und ISiK MedizinischeBehandlungseinheit (analog zu 2.0.5 und 4.0.0-rc2) https://github.com/gematik/spec-ISiK-Terminplanung/pull/199/commits/7218c4f9f085604e311761d5e76647d09aea04d1
```

**Review by alexey-tschudnowsky — DISMISSED** [review]

File: ImplementationGuide/markdown/ReleaseNotes.md [hunk 4]
```diff
@@ -4,6 +4,14 @@ Im Rahmen der ISiK-Veröffentlichungen wird das [Semantic Versioning](https://se
 
 Die erste Ziffer X bezeichnet ein Major-Release und regelt die Gültigkeit von Releases. Die dritte Ziffer Y (Release x.0.y) bezeichnet eine technische Korrektur und versioniert kleinere Änderungen (Packages) während eines Jahres, z. B. 1.0.1.
 
+Version: 3.0.5
+
+Datum: tbd.
```
Comment (f-peverali) [inline ImplementationGuide/markdown/ReleaseNotes.md]:
```suggestion
Datum: 03.07.2024
```

**Pull request merged**